wsi_deid/import_export.py

```diff
@@ -110,9 +110,32 @@ def getSchema():
     # that contain a single file that parses as valid json, then we need to
     # load those files and combine them into a single schema in lieu of using
     # this default SCHEMA_FILE_PATH
-    logger.info('Using %s (length %d) for schema' % (
-        SCHEMA_FILE_PATH, os.path.getsize(SCHEMA_FILE_PATH)))
-    return json.load(open(SCHEMA_FILE_PATH))
+    schemaFolderId = Setting().get(PluginSettings.WSI_DEID_SCHEMA_FOLDER)
+    mergedSchema = {}
+
+    if schemaFolderId:
+        schemaFolder = Folder().load(schemaFolderId, force=True)
+
+        for filepath, file in Folder().fileList(schemaFolder, data=False):
+            currentObject=json.load(File().open(file))
+            print(type(currentObject))
+            for key, value in currentObject.items():
```

Comment:
I think we have found and successfully decoded multi schemas, we can just combine them via
{'oneOf': [<list of loaded schemas>]}` and not do any sort of dictionary/key merging ourselves.  I'd probably only add this if the list is longer than 1 (and if it is zero, we'd fall back to the original schema file path).

Reply:
We might need to add the schema specification in to that as well (but I'd try it without doing so first).  That is, maybe
`{"$schema": "http://json-schema.org/draft-07/schema", "oneOf": [<list of loaded schemas>]}` might be needed.

Reply:
added in 2a93feaf7fb521edcaeb54d9812d4320d10df69e